docker-compose.yml

```diff
@@ -25,7 +29,10 @@ services:
   db:
     container_name: protectedplanet-db
     image: kartoza/postgis:11.5-2.5
-    network_mode: host
+    ports:
+      - "5432:5432"
```

Comment:
Please change to "5432" (without host port), so it won't conflict with host port. This is Jenkins test, no one will connect from the host.